tests/Utils.ts

```diff
@@ -157,6 +157,25 @@ export class TestAgora implements FullNodeAPI
             }, next);
         });
 
+        this.agora.get("/merkle_path", (req: express.Request, res: express.Response, next) =>
+        {
+            if (req.query.block_height === undefined || Number.isNaN(req.query.block_height))
+                return Promise.reject(new Error('Missing or invalid query parameter block_height'));
+
+            if (req.query.hash === undefined)
+                return Promise.reject(new Error('Missing query parameter enroll_key'));
```

Comment:
```suggestion
            if (req.query.hash === undefined)
            {
                res.status(400).json({ statusText: 'Missing or invalid hash query parameter' });
                return;
            }
```

Reply:
Thank you for the suggestion. Should this comment be removed?
https://github.com/bosagora/stoa/blob/534ef096aa1742749cc39d231b1fcfd20f4da494/tests/Utils.ts#L107